core/controllers/topic_editor.py

```diff
@@ -229,7 +240,14 @@ def put(self, topic_id):
             raise self.InvalidInputException(e)
 
         topic_dict = topic_services.get_topic_by_id(topic_id).to_dict()
-
+        topic_dict['uncategorized_skill_descriptions'] = (
```

Comment:
Same thing -- it doesn't seem like good design to modify the topic dict like this.

Reply:
Done.